src/models/level/index.js

```diff
@@ -28,7 +28,10 @@ const ClassSpecific = new Schema({
   bardic_inspiration_die: { type: Number, index: true },
   brutal_critical_dice: { type: Number, index: true },
   channel_divinity_charges: { type: Number, index: true },
-  creating_spell_slots: [ClassSpecificCreatingSpellSlot],
+  creating_spell_slots: {
+    type: [ClassSpecificCreatingSpellSlot],
+    default: undefined,
```

Comment:
Holy shit. I didn't know you could do this. This is actually a bug in all the other places we use arrays in nested objects.

Reply:
Do you want to do the others while you're in here?

Reply:
sure! i'll tidy the rest of them up this week sometime. we can either merge this pr now or leave it open and i can tack on fixes for any other instances i find. either way is fine by me, it's a super low impact bug so not too worried about it being there for a bit longer.

Reply:
I'll just leave it open for now.